public/app/features/dashboard/components/PanelEditor/PanelEditor.tsx

```diff
@@ -131,11 +131,16 @@ export class PanelEditorUnconnected extends PureComponent<Props> {
       return;
     }
 
-    const targetPane = pane === Pane.Top ? 'topPaneSize' : 'rightPaneSize';
     const { updatePanelEditorUIState } = this.props;
-    updatePanelEditorUIState({
-      [targetPane]: size,
-    });
+    if (pane === Pane.Top) {
+      updatePanelEditorUIState({
+        topPaneSize: `${(size / window.innerHeight) * 100}%`,
+      });
+    } else {
+      updatePanelEditorUIState({
+        rightPaneSize: size,
```

Comment:
Would be awesome to have percentages here as well to be aligned with top pane sizing.

Reply:
I wasn't precise enough - instead of percentages I have meant pane size/ window size ratio